JoystickMouseControl.ino

```diff
@@ -0,0 +1,120 @@
+/*
+  JoystickMouseControl
+
+  Controls the mouse from a joystick on an Arduino Leonardo, Micro or Due.
+  Uses a pushbutton to turn on and off mouse control, and a second pushbutton
+  to click the left mouse button.
+
+  Hardware:
+  - 2-axis joystick connected to pins A0 and A1
+  - pushbuttons connected to pin D2 and D3
+
+  The mouse movement is always relative. This sketch reads two analog inputs
+  that range from 0 to 1023 (or less on either end) and translates them into
+  ranges of -6 to 6.
+  The sketch assumes that the joystick resting values are around the middle of
+  the range, but that they vary within a threshold.
+
+  WARNING: When you use the Mouse.move() command, the Arduino takes over your
+  mouse! Make sure you have control before you use the command. This sketch
+  includes a pushbutton to toggle the mouse control state, so you can turn on
+  and off mouse control.
+
+  created 15 Sep 2011
+  updated 28 Mar 2012
+  by Tom Igoe
+
+  This example code is in the public domain.
+
+  http://www.arduino.cc/en/Tutorial/JoystickMouseControl
+*/
+
+#include "Mouse.h"
+
+// set pin numbers for switch, joystick axes, and LED:
+const int switchPin = 2;      // switch to turn on and off mouse control
+const int mouseButton = 3;    // input pin for the mouse pushButton
```

Comment:
```suggestion
const int mouseButton = 3;    // input pin for the mouse pushbutton
```
The camelCase makes this seem like it's referring to a variable. The term "pushbutton" is used in the other comments.